tests/function_libs/torch_lib/ops_test_common.py

```diff
@@ -56,7 +57,8 @@
 )
 
 TEST_OPSET_VERSION = 18
-IS_WINDOWS = os.name == "nt"
+IS_MACOS = sys.platform.startswith('darwin')
+IS_WINDOWS = sys.platform.startswith('win32')
```

Comment:
I think the original check for windows is reliable; or are there exceptions not handled?

Reply:
Had changed the format for the sake of uniformity. But the previous check was sufficient. Reverting back to that.